samples/snippets/cpp/VS_Snippets_CLR_Classic/classic NotifyFilters Example/CPP/source.cpp

```diff
@@ -1,70 +1,89 @@
-// <Snippet1>

```

Comment:
@gewarren do you know why the examples are not visible in the MS Docs preview? Is it because I removed the `<Snippet1></Snippet1>`?

Reply:
Yes, the #1 at the end of the line means "Snippet1".

https://github.com/carlossanlop/dotnet-api-docs/blob/ed157b8a6c8c714551b2c4919877269c8f6c1f38/xml/System.IO/FileSystemWatcher.xml#L129

Reply:
If you just want to show the whole file, there's a way to do that. See the first example here: https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#in-repo-snippet-references